client/src/utils/media.ts

```diff
@@ -0,0 +1,17 @@
+import '@expo/match-media';
+
+import { useMediaQuery } from 'react-responsive';
+
+export type MediaQueryType = {
+  tablet: boolean,
+  desktop: boolean,
+}
+
+export const useMedia = (): MediaQueryType => {
+  const isTablet = useMediaQuery({ minWidth: 768 });
+  const isDesktop = useMediaQuery({ minWidth: 1224 });
+  return {
+    tablet: isTablet && !isDesktop,
+    desktop: isDesktop,
```

Comment:
Can we have `mobile` and `tablet` here? I'd like to consider `mobile` as our first citizen and rests as `> tablet` in first hand!

Reply:
> I'd like to consider mobile as our first citizen

@hyochan That's exactly why I chose `tablet` and `desktop` and left out `mobile` here.
```ts
const StyledScrollView = styled.ScrollView`

  width: 100%;  // Styles are for mobile by default (first-citizen)

  // Override for tablet
  ${({ theme: { tablet } }) => tablet && css`
    width: 75%;
  `}

  // Override for desktop
  ${({ theme: { desktop } }) => desktop && css`
    width: 50%;
    max-width: 800;
  `}
`;
```

Reply:
What if we have only `tablet` here? This isn't `> tablet` since I see `isTablet && !isDesktop`.
Originally, media query overrides other bigger size screens if it reaches out last big screen which here would be the `tablet`.

Reply:
Also, I want to order `rebasing` since there is a conflict :)